bin/install.sh

```diff
@@ -14,20 +13,57 @@ cp $HOME/stanford_travisci_scripts/aliases.drushrc.php $HOME/.drush/aliases.drus
 sed -ie "s|HOME|$HOME|" $HOME/.drush/aliases.drushrc.php
 cat $HOME/.drush/aliases.drushrc.php
 
-if [ -z "$PRODUCT_NAME" ]; then
-  if [ ! -z "$DRUPAL_PROFILE_BRANCH" ]; then DRUPAL_PROFILE_BRANCH="-b $DRUPAL_PROFILE_BRANCH"; fi
-  git clone --depth 1 $DRUPAL_PROFILE_BRANCH https://github.com/SU-SWS/Stanford-Drupal-Profile.git $HOME/Stanford-Drupal-Profile
-  grep -rl 'git@github.com:' $HOME/Stanford-Drupal-Profile | xargs sed -i 's|git@github.com:|https://github.com/|'
-  drush make -y --force-complete $HOME/Stanford-Drupal-Profile/make/dept.make $HOME/html
-  drush @local si -y stanford --db-url=mysql://root@localhost/drupal --account-name=admin --account-pass=admin
-else
+function install_from_deployer {
```

Comment:
For the sake of code clarity you should move your functions to a separate include file. This will make the reading the order of execution much easier.